### ISSUE TITLE
Investigate compressed CSV files

### ISSUE THREAD

**Issue by ngreenwald** [angelolab/ark-analysis#775]
**Is your feature request related to a problem? Please describe.**
The cell table CSV files can get quite large and unwieldy. 

**Describe the solution you'd like**
Investigate the default compression options in pandas for saving CSVs. If they seem acceptable, use that for the segmentation notebook (and elsewhere) to reduce file size.

**Comment by srivarra** [comment]
Pandas' `to_csv` has options for compression. I tried [zstd](http://facebook.github.io/zstd/) with [`python-zstandard`](https://python-zstandard.readthedocs.io/en/latest/index.html), and [zip](https://docs.python.org/3/library/zipfile.html).

| Algorithm | Original File Size (Gb) | Compressed File Size (Gb) | Time (min) | Compression Ratio |
|-----------|-------------------------|---------------------------|------------|-------------------|
| ZIP       | 3.3                     | 1.4                       | 4          | 2.4:1             |
| ZSTD      | 3.3                     | 1.3                       | 1.5        | 2.5:1             |
| ZIP       | 9.3                     | 3.9                       | 12         | 2.4:1             |
| ZSTD      | 9.3                     | 3.7                       | 5          | 2.5:1             |


In summary zstd is fast and gives a good compression ratio. The only issue is that in order to view the contents of the cell table, it must be decompressed (pandas can do this in `read_csv`, but Excel can't decompress it).

**Comment by ngreenwald** [comment]
Can all of the other methods still be opened with Excel?

**Comment by srivarra** [comment]
No method will allow for compressed csv's to be opened with Excel.

**Comment by ngreenwald** [comment]
Okay got it. In that case, will you expose the compress option to the user in the segmentation notebook, with a note saying it'll make the files smaller but prevent opening directly in excel
